src/data/teams.json

```diff
@@ -762,6 +762,19 @@
           "profileImage": "https://ca.slack-edge.com/T07PJD1FZ-U01HR3S3WCS-810bc78e23bd-512",
           "slackUsername": "Chatrasal Singh"
         }
+      },
+      {
+        "title": "Marketing Analyst",
+        "isLead": false,
+        "payPerDay": 2800,
+        "createdDate": "2020-07-08T02:14:59.307535Z",
+        "contributor": {
+          "contributorId": "U01JJ677TL0",
```

Comment:
the contributor id is incorrect, please refer to the `last contributor id + 1`

Reply:
Alright.